packages/forms-web-app/src/pages/examination/exmaination.router.js

```diff
@@ -15,7 +21,19 @@ const { decodeUri } = require('../../middleware/decode-uri');
 const { projectsMiddleware } = require('../projects/_middleware/middleware');
 const {
 	addCommonTranslationsMiddleware
-} = require('../../../src/pages/_middleware/i18n/add-common-translations-middleware');
+} = require('../../pages/_middleware/i18n/add-common-translations-middleware');
+const {
```

Comment:
I'm thinking that it might be better if we just have one translation file for the entire journey as there is not a large amount of content for each page and we would assume that someone on the journey would hit almost every page. Then we can reduce the need for all the add translations middleware functions.